components.json

```
@@ -1346,6 +1346,11 @@
 			"require": "markup",
 			"owner": "Golmote"
 		},
+		"wolfram": {
+			"title": "Wolfram language",
+			"alias": ["wl", "mathematica"],
```

Why did you remove the alias title? It was really good.

```suggestion
			"alias": ["wl", "mathematica"],
			"aliasTitles": {
				"mathematica": "Mathematica"
			},
```

---

Ah sorry, adding them back.

---

No need to apologize. It's not like I'm your boss lol